pywps/configuration.py

```diff
@@ -149,9 +149,9 @@ def load_configuration(cfgfiles=None):
         cfgfiles = [cfgfiles]
 
     if 'PYWPS_CFG' in os.environ:
-        config_files.append(os.environ['PYWPS_CFG'])
+        cfgfiles.append(os.environ['PYWPS_CFG'])
```

Comment:
Oh wow, good catch ! Without this fix, the default config file was completely ignored if `PYWPS_CFG` is set?

Reply:
The service did not start when `PYWPS_CFG` was set.